app/models/deletion.rb

```diff
@@ -16,7 +16,7 @@ class Deletion < ActiveRecord::Base
   private
 
   def version_is_indexed
-    errors.add(:number, "is already deleted") unless @version.indexed?
+    errors.add(:base, "#{rubygem_name} #{version} is already deleted") unless @version.indexed?
```

Comment:
Looking good! Let's make this match the old message: "has already been deleted" 👍